### DISCUSSION TITLE
Does NeMo support Language Model for ASR task?

### DISCUSSION THREAD

**Discussion by nghiahuynh-ai** [NVIDIA-NeMo/NeMo#5295] [General]
I have used NeMo framework for months, it's really tasty to me. But it does not seem to support Language Model for ASR task. Let me know if I am wrong as well as it really does not. Thanks in advance.

**Comment by VahidooX** [comment]
We support two language modelling approaches: 1-N-gram language modelling 2-Neural rescoring

Here is the doc page:
https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/asr_language_modeling.html
